cmd/nitro/nitro.go

```diff
@@ -241,10 +241,14 @@ func mainImpl() int {
 	if nodeConfig.Execution.Sequencer.Enable && !nodeConfig.Execution.Sequencer.Dangerous.Timeboost.Enable && nodeConfig.Node.TransactionStreamer.TrackBlockMetadataFrom != 0 {
 		log.Warn("Sequencer node's track-block-metadata-from is set but timeboost is not enabled")
 	}
-	if nodeConfig.Node.EspressoCaffNode.Enable && (nodeConfig.Execution.Sequencer.Enable || nodeConfig.Node.Sequencer || nodeConfig.Node.SeqCoordinator.Enable || nodeConfig.Node.DelayedSequencer.Enable) {
+	if nodeConfig.Node.EspressoCaffNode.Enable && (nodeConfig.Execution.Sequencer.Enable || nodeConfig.Node.Sequencer || nodeConfig.Node.SeqCoordinator.Enable || nodeConfig.Node.DelayedSequencer.Enable || nodeConfig.Execution.Sequencer.Dangerous.Timeboost.Enable) {
 		log.Error("The Caff node cannot have any type of sequencer enabled, run without a config that enables any form of sequencer.")
 		return 1
 	}
+  if nodeConfig.Node.EspressoCaffNode.Enable && len(nodeConfig.Execution.ForwardingTarget) == 0{
```

Comment:
Forwarding target is usually set to null
  "forwarding-target": "null"
  would our code panic here if it is null?

Reply:
`nodeConfig.Execution.ForwardingTarget` is a variable of type `string` which [cannot be nil/null](https://stackoverflow.com/a/36006313) in go.

If any of the variables in the config aren't set, they default to the values in `ConfigDefault` in `arbnode/node.go`, or they default to that variables types default value. In the case of `string`, this default is `""`